scripts/jenkins/ardana/manual/deploy-crowbar.sh

```diff
@@ -20,7 +20,7 @@ source lib.sh
 
 validate_input
 setup_ansible_venv
-mitogen_disable
+mitogen_enable
```

Comment:
Why is this here?
Recent discussions sounded like it has problems. Do you know of any? Maybe this warrants its own PR.
This commit has more code changes that I wouldn't have expected. The commit message sounds like it is only supposed to change documentation. Were those intentional?

Reply:
This change got merged in the main branch, so it's not in this PR anymore.